src/pages/settings/Profile/CustomStatus/SetTimePage.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import FullscreenLoadingIndicator from '../../../../components/FullscreenLoadingIndicator';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import HeaderWithBackButton from '../../../../components/HeaderWithBackButton';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import TimePicker from '../../../../components/TimePicker';
+import Form from '../../../../components/Form';
+import usePrivatePersonalDetails from '../../../../hooks/usePrivatePersonalDetails';
+import useLocalize from '../../../../hooks/useLocalize';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import * as User from '../../../../libs/actions/User';
+import DateUtils from '../../../../libs/DateUtils';
+import * as ValidationUtils from '../../../../libs/ValidationUtils';
+import compose from '../../../../libs/compose';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import ROUTES from '../../../../ROUTES';
+import styles from '../../../../styles/styles';
+
+const propTypes = {
+    /** User's private personal details */
+    privatePersonalDetails: PropTypes.shape({
+        dob: PropTypes.string,
```

Comment:
Same here, I don't think we're using this!

Reply:
👍🏻